### ISSUE TITLE
[assignment 6] is there no restriction in codes?

### ISSUE THREAD

**Issue by 7dy109** [kaist-cp/cs220#125]
In assignment 6, is there no restriction in codes which means If I see "Yes" in assignment 6 in gg, then I will get a full score in this assignment?
Thank you for your answer.

**Comment by zzz845zz** [comment]
Yes. But please do not use the trick to pass only the test cases: https://github.com/kaist-cp/cs220/issues/63#issuecomment-1257437543.

**Comment by 7dy109** [comment]
Thank you very much!!!!!!
